app/ui/setup.py

```diff
@@ -59,6 +59,46 @@ def setup_block():
         ).values[0]
     )
 
+    req = perms.request_aws_api_integration(
+        "opscenter_api_integration",
+        (
+            "https://1lf9af4dk7.execute-api.us-east-1.amazonaws.com",
+            "https://mr2gl3hcuk.execute-api.us-east-2.amazonaws.com",
+            "https://1fb567sika.execute-api.us-west-2.amazonaws.com",
+            "https://rkb9hwsqw0.execute-api.us-east-1.amazonaws.com",
+            "https://hh538sr9qg.execute-api.us-west-2.amazonaws.com",
+            "https://w4cu711jd2.execute-api.us-west-2.amazonaws.com",
+        ),
+        perms.AwsGateway.API_GATEWAY,
+        "arn:aws:iam::323365108137:role/SnowflakeOpsCenterRole",
+        None,
+        "OPSCENTER_SUNDECK_EXTERNAL_FUNCTIONS",
+        None,
+    )
+    if req is None:
+        print("Token recorded, creating API integration.")
+        st.code(
```

Comment:
I need help on this. 

1. On accessing reference I see "missing a reference association". Code is similar to  [this](https://github.com/sundeck-io/OpsCenter/blob/16379651411256033c7121c44325a67d9fdf04b1/app/ui/setup.py#L138). This if is success case. See Screenshot

![Screenshot 2023-07-18 at 3 21 31 PM](https://github.com/anshuldata/OpsCenter/assets/42910861/6871c384-0de6-47f2-9a45-84c889203006)